spec/parliament/request/base_request_spec.rb

```diff
@@ -61,10 +61,10 @@
     end
 
     context 'it accepts query parameters' do
```

Comment:
What is this actually testing?

How does this test prove that what you're expecting to happen, actually happens? You don't appear to be checking that the params hash actually generates the URL you expect? Just that a connection to that URL could be made. Infact, this response isn't even checking that it's a 200?